### ISSUE TITLE
Specified part does not exist in the package

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode-vsce#482]
I'm publishing an extension with `--web` but kept seeing error `Specified part does not exist in the package`. Only after removing a version file:

File name: `out/version`

File content:

```txt
ebe71405e07d4ae16808e7e05f94fa29c53c142b

```

The error is gone.

**Comment by sandy081** [comment]
It seems like Marketplace does not like to create an asset for a file that does not has extension

**Comment by homeboy445** [comment]
@joaomoreno Thanks for responding! Here are the steps I followed:
1.) I generated my extension using yo code.
2.) Made my changes.
3.) Then I provided my PAT to the `vsce login` and it succeeded.
4.) Afterwards that, I went with the command `vsce publish` which showed me `ERROR: Specified part does not exist in the package`.
I tried changing several options in my package.json but got no luck with that. Is there something I'm missing which is leading to this behaviour or is there something else? Thanks a lot for helping!
